authors/tests/test_tags.py

```diff
@@ -0,0 +1,47 @@
+import json
+
+from rest_framework.views import status
+from django.urls import reverse
+
+# local imports
+from .base_test import TestBase
+
+
+class TagCreation(TestBase):
+    """Test if a tag list is created"""
+
+    def test_tag_addition(self):
+        """Test tags addition to article"""
+        token = self.authentication_token()
+        self.client.credentials(HTTP_AUTHORIZATION='Bearer ' + token)
+        self.client.get(self.get_verify_url(self.user_data))
+        response = self.client.post(
+            self.article_url,
+            data=json.dumps(self.valid_taglist_data),
+            content_type='application/json'
+        )
+        length_of_taglist = len(response.data['tagList'])
+        self.assertEqual(length_of_taglist, 2)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def test_tag_update(self):
+        """Test tags update"""
+        token = self.authentication_token()
+        self.client.credentials(HTTP_AUTHORIZATION='Bearer ' + token)
+        self.client.get(self.get_verify_url(self.user_data))
```

Comment:
Lines 29 to 31 are handled by the `verify_user` method in the `TestBase` class, please use that instead of repeating them.